CHANGELOG.md

```diff
@@ -57,6 +57,15 @@ and this project adheres to Rust's notion of
   - `Builder::{spends, outputs}`
   - `SpendError`
   - `OutputError`
+- `orchard::keys`:
+  - `PreparedEphemeralPublicKey`
+  - `PreparedIncomingViewingKey`
+- impls of `memuse::DynamicUsage` for:
+  - `orchard::note::nullifier::Nullifier`
```

Comment:
```suggestion
  - `orchard::note::Nullifier`
```

Reply:
Oh, I looked at the code and it is actually defined in `orchard::note::nullifier::Nullifier`, but reexported at `orchard::note::Nullifier`, which is why I changed it. I guess it doesn't matter either way, this is fine.

Reply:
The `nullifier` module is not public, so to users of this crate it is never part of the module path.